### DISCUSSION TITLE
osu! Faults occur when shrinking the circle in the model

### DISCUSSION THREAD

**Discussion by iam6668** [ppy/osu#26453] [Q&A] [answered]
Due to my low language proficiency, I really can't find a suitable place to report this problem, nor can I find out the root cause of the problem, let alone find out whether there are feedback on similar problems. I can only use translation software to explain this problem in the discussion area as much as possible.
![~08(CKK$}2Q`Q P1P4W}W J](https://github.com/ppy/osu/assets/68184937/7131443b-a189-46da-9c9b-f81173cf7612)
As shown in the figure, when shrinking the circle during the game, the outer circle image fault often occurs, and the frequency is at least 30% (on a map). This is the case no matter what skin is used. During this period, I tried various methods such as updating the driver, restarting, reinstalling, switching rendering methods, etc., but this has been the case since at least November, and as of the latest version, this problem still exists.
What’s even weirder is that when I use some recording software (such as obs starting recording, or QQ screen sharing), this situation almost never happens (of course it still happens with a small probability), and this picture is taken with a mobile phone. , because this situation will disappear once you start recording
I hope someone will answer it as soon as possible. I have put the log here. If you need more feedback, please tell me as soon as possible. I will put some information at the end.
[compressed-logs.zip](https://github.com/ppy/osu/files/13869986/compressed-logs.zip)

Windows 10, osu! , the latest version so far (i.e. version 20231231)
Note: QQ is a chat software

The following is the original text. If any Chinese people see any translation problems, please correct them. I would be very grateful.
由于语言水平不高，我实在找不到合适的地方去反馈这个问题，也无法找出问题根源，更没有办法寻找是否有类似问题反馈，只能尽可能用翻译软件在讨论区说明这个问题
如图所示，在游玩过程中缩圈时经常出现外圈图像断层的情况，频率至少在三成以上（在一张地图上），无论用什么皮肤都是这种情况。在这期间我尝试了更新驱动，重启，重新安装，切换渲染方式等各种办法，但至少从11月开始一直都有这种情况，截止最新版本依旧会有这个问题。
更奇怪的是当我在使用一些录制软件时（比如obs开始录制，或者QQ屏幕共享时），几乎就不会出现这样的情况（当然仍然小概率发生），其中这张图片就是用手机拍摄屏幕，因为一旦开始录制这个情况就会消失
希望尽快有人回答吧，日志我放在这里了，如果需要更多反馈请尽快跟我说，最后放些信息
Windows10，osu！，截至目前最新版本（即20231231版本）
备注，QQ为一款聊天软件

以下是原文，如果有中国人看到发现翻译的问题，也请指正，感激不尽

**Comment by peppy** [comment, marked as answer]
This is called screen tearing. Turn on vsync.
